models/tasktemplate/fromdirs.go

```diff
@@ -17,30 +17,33 @@ var (
 	discoveredTemplates []TaskTemplate = []TaskTemplate{}
 )
 
-// LoadFromDir reads yaml-formatted task templates
+// LoadFromDirs reads yaml-formatted task templates
```

Comment:
I prefer the keep this functions as-is and split the paths elsewhere

Reply:
agreed